client/context/globalContext.tsx

```diff
@@ -9,15 +9,12 @@ const initialDataContext: DataContextProps = {
   user: null,
   setUser: () => {},
   profiles: [],
-  setProfiles: () => [],
+  setProfiles: () => {},
```

Comment:
this should be a list, no?

Reply:
its a function : 
  profiles: ProfileType[];
  setProfiles: React.Dispatch<React.SetStateAction<ProfileType[]>>;